components/Accordion.vue

```diff
@@ -6,41 +6,60 @@
     name="benefit-list"
     tag="div"
     aria-multiselectable="true"
-    aria-live="polite">
+    aria-live="polite"
+  >
     <div
       v-for="benefit in lifeEventBenefits"
       :key="`acc-key-${benefit.slug}`"
-      class="break-inside-avoid margin-bottom-2">
+      class="break-inside-avoid margin-bottom-2"
+    >
```

Comment:
a lot of prettier formatting

Reply:
I think I had a PR that was funny too... I thought it was a me problem. 
Maybe we need a huddle to make sure we all have our IDEs configured correctly and the run prettier against entire project worked as expected.   Im not sure what we missed because when we went through that everything seemed to work as expected.

Reply:
Sounds like a good idea!